packages/cli/src/workflows/workflow.service.ts

```diff
@@ -202,15 +203,17 @@ export class WorkflowService {
 			]),
 		);
 
-		if (tagIds && !config.getEnv('workflowTagsDisabled')) {
+		const tagsDisabled = Container.get(GlobalConfig).tags.workflowTagsDisabled;
```

Comment:
we should add `GlobalConfig` to the constructor, and replace this with 
```suggestion
		const tagsDisabled = this.globalConfig.tags.workflowTagsDisabled;
```